client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/entities/SchemaString.java

```diff
@@ -47,7 +49,15 @@ public void setSchemaString(String schemaString) {
     this.schemaString = schemaString;
   }
 
-  public String toJson() throws IOException {
+  public String toJson(boolean pretty) throws IOException {
```

Comment:
I don't think I need this
```suggestion
  public String toJson() throws IOException {
```